libs/labelbox/src/labelbox/client.py

```diff
@@ -751,6 +751,8 @@ def create_project(self, **kwargs) -> Project:
             queue_mode (Optional[QueueMode]): The queue mode to use
             quality_mode (Optional[QualityMode]): The quality mode to use (e.g. Benchmark, Consensus). Defaults to
```

Comment:
can we document that this attr will be deprecated in a future version. And also print a deprecation version if this attr is used